content/en/docs/Configuration/kialis.kiali.io.md

```diff
@@ -4423,6 +4424,25 @@ to <code>secret:myGrafanaCredentials:myGrafanaPw</code>.</p>
 </div>
 </div>
 
+<div class="property depth-5">
+<div class="property-header">
+<hr/>
+<h3 class="property-path" id=".spec.external_services.istio.gateway_api_classes_label_selector">.spec.external_services.istio.gateway_api_classes_label_selector</h3>
+</div>
+<div class="property-body">
+<div class="property-meta">
+<span class="property-type">(string)</span>
+
+</div>
+
+<div class="property-description">
+<p>Specifies the label selector for automatic discovery of K8s Gateway API Classes. This setting is used when the <code>gateway_api_classes</code> configuration is not explicitly defined. It defaults to empty, which means all Gateway API Classes will be loaded.</p>
+
+</div>
+
+</div>
+</div>
+
```

Comment:
Revert all changes to this file. This file is auto-generated from the CRD Schema that you modified in the operator PR.